Okta.AspNetCore/GlobalSuppressions.cs

```diff
@@ -1,7 +0,0 @@
-﻿// <copyright file="GlobalSuppressions.cs" company="Okta, Inc">
-// Copyright (c) 2018-present Okta, Inc. All rights reserved.
-// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
-// </copyright>
-
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA1600:Elements should be documented", Justification = "Reviewed")]
-[assembly: System.Diagnostics.CodeAnalysis.SuppressMessage("StyleCop.CSharp.DocumentationRules", "SA0001XmlCommentAnalysisDisabled", Justification = "Reviewed")]
```

Comment:
It seems like we are removing these suppressions by adding documentation to all the public members, is that right? If that's the case we should probably note that in the PR description, since this is not directly related to upgrading to .NET core 3.0.

Reply:
You're right. I removed those suppressions since I fixed those warnings.